server.go

```diff
@@ -40,6 +40,8 @@ import (
 
 var logger *logging.Logger
 
+const tcpReadTimeout time.Duration = 59 * time.Second
```

Comment:
Could you add a comment explaining why 59 seconds?

Reply:
done